Gulpfile.js

```diff
@@ -8,12 +8,18 @@ gulp.task('clean', function () {
     return del('lib');
 });
 
-gulp.task('build', ['clean', 'lint'], function () {
+gulp.task('build', ['lint', 'scripts'], function () {
     return gulp.src('src/**/*.js')
         .pipe(babel())
         .pipe(gulp.dest('lib/'));
 });
 
+gulp.task('scripts', ['clean'], function () {
+    return gulp
+        .src(['src/files/*.bat'])

```

Comment:
Task name `scripts` and folder name `files` are not suitable. Because all other files are `files` and `scripts` too.

I propose call the folder `prerun` and call the gulp task `prerun-scripts`